fontforge/lookups.c

```diff
@@ -4701,6 +4701,8 @@ static void GrowBufferAddClass(GrowBuf *gb,int class_n,char **classnames, int cl
 int GlyphNameCnt(const char *pt) {
     int cnt = 0;
 
+    /* Avoid a segfault if something is done wrong. */
+    if (!pt) { return 0; }
```

Comment:
It would be better to avoid calling this function with a null pointer than adding this check.